### ISSUE TITLE
[1.16.5] Delightful Dirt doesn't spawn rabbits?

### ISSUE THREAD

**Issue by Kaligraphy22** [vadis365/Mob-Grinding-Utils#202]
1.16.5-forge-36.2.29
mob_grinding_utils-1.16.5-0.4.35.jar

What's odd about this is that the Delightful Dirt spawns racoons from Alex's Mobs and the sylph, drygmy and carbuncles from Ars Novueau... but not a passive mob from vanilla Minecraft. I get every other passive vanilla mob - cows, sheep, pigs, chickens and horses - but no rabbits. Not sure if this is a bug or I'm overlooking something.
![2022-05-04_19 07 47](https://user-images.githubusercontent.com/72703227/166845441-5821f84e-57ec-4754-8137-2744e063565f.png)
.

**Comment by russfort** [comment]
Is it a biome issue?

**Comment by Flanks255** [comment]
So while i cant find it in code, various wiki's show rabbits spawn in certain biomes, and quick testing confirms this, i put delightful dirt in a taiga biome, and boom rabbits, foxes too lol.
Interestingly, there is a blocktag for Rabbit_spawnable_on, and foxes_spawnable_on, but they spawn no matter if i apply the tag or not, as long as its in the right biome.

**Comment by Kaligraphy22** [comment]
Wow okay, thanks for this cause I never once checked to see WHAT biomes rabbits actually spawned in. I just assumed "Hey, this is my test world, which is a flat world so it's a plains biomes. Yup, ordinary. And if the rest of the regular mobs spawn, surely rabbits should." I just checked the wiki and realized my own foolishness. Rabbits don't even spawn in Plains! And for all the times I've used this mod, I never once realized it was respecting the biome's spawn list. Oi, I'm a fool. And there's a taiga near my base so problem solved. We can close this issue now.
